jenkins-integration/jenkins-jobs/job_bootstrap.groovy

```diff
@@ -58,8 +58,8 @@ class DSLHelper {
 // accomplish the same by simply modifying the values in the Jenkins gui
 // on the 'configuration' screen for the seed job.
 
-def git_repo = 'https://github.com/puppetlabs/gatling-puppet-load-test.git'
-def git_branch = 'master'
+def git_repo = 'https://github.com/magisus/gatling-puppet-load-test.git'
+def git_branch = 'empty-repo-5x'
```

Comment:
Is this still needed? With the latest changes do we expect it to succeed (IIRC, there's maybe still an issue with a hiera file?)

Reply:
I didn't mean to commit this. This was only needed for dev testing.